### PR TITLE
Convert user sub-lists to tables

### PR DESCRIPTION
**Pull request by keithjgrant**
##### SUMMARY

Converts User Organizations, Teams, and Roles lists to tables

Addresses #6189

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
 - UI

### CODE REVIEW

**Comment by AlexSCorey** [comment]
E2E Tests triggered

@keithjgrant noticed this when sorting by name on User --> Roles

Screen Shot 2021-04-30 at 11 34 20 AM

**Comment by AlexSCorey** [comment]
On User --> Organizations if I have an organization named `New Org` and search for `New` nothing is returned.  However, adding``` toolbarSearchColumns={[
        {
          name: t`Name`,
          key: 'name__icontains',
          isDefault: true,
        },
      ]} ```
to the `<PaginatedTable>` inside `UserOrgnaizationList.jsx` solves this problem.

**Comment by AlexSCorey** [comment]
The User --> Roles list is confusing to me.  I think this has been discussed and IIRC we have to lay it out this way due to limitations from from the api/pagination.  However, it would be great if the Roles and Name column were switched because it seems that the roles are actually the subject of this list instead of the name of the resource the roles are applied to.

File: awx/ui_next/src/screens/User/UserRoles/UserRolesListItem.test.jsx [hunk 1]
```diff
@@ -19,63 +19,91 @@ describe('<UserRolesListItem/>', () => {
...
     );
+    const cells = wrapper.find('Td');
+    expect(cells.at(0).text()).toBe('template delete project');
     expect(
-      wrapper.find('PFDataListCell[aria-label="Resource name"]').text()
-    ).toBe('template delete project');
-    expect(
-      wrapper.find('PFDataListCell[aria-label="Resource type"]').text()
+      cells.at(1).text()
+      // wrapper.find('PFDataListCell[aria-label="Resource type"]').text()
```
Comment (AlexSCorey) [inline awx/ui_next/src/screens/User/UserRoles/UserRolesListItem.test.jsx]: Can this be removed?

File: awx/ui_next/src/screens/User/UserRoles/UserRolesListItem.test.jsx [hunk 2]
```diff
@@ -19,63 +19,91 @@ describe('<UserRolesListItem/>', () => {
...
     ).toContain('Job Template');
     expect(
-      wrapper.find('PFDataListCell[aria-label="Resource role"]').text()
+      cells.at(2).text()
+      // wrapper.find('PFDataListCell[aria-label="Resource role"]').text()
```
Comment (AlexSCorey) [inline awx/ui_next/src/screens/User/UserRoles/UserRolesListItem.test.jsx]: Can this be removed?

File: awx/ui_next/src/screens/User/UserRoles/UserRolesListItem.test.jsx [hunk 3]
```diff
@@ -19,63 +19,91 @@ describe('<UserRolesListItem/>', () => {
...
+        </tbody>
+      </table>
     );
     expect(
-      wrapper.find('PFDataListCell[aria-label="Resource name"]').text()
+      wrapper
+        .find('Td')
+        .at(0)
+        .text()
+      // wrapper.find('PFDataListCell[aria-label="Resource name"]').text()
```
Comment (AlexSCorey) [inline awx/ui_next/src/screens/User/UserRoles/UserRolesListItem.test.jsx]: Can this be removed?

File: awx/ui_next/src/screens/User/UserRoles/UserRolesList.jsx [hunk 4]
```diff
@@ -121,41 +123,46 @@ function UserRolesList({ i18n, user }) {
...
-            name: i18n._(t`Role`),
+            name: t`Role`,
             key: 'role_field__icontains',
             isDefault: true,
           },
         ]}
         toolbarSortColumns={[
           {
-            name: i18n._(t`ID`),
+            name: t`ID`,
```
Comment (AlexSCorey) [inline awx/ui_next/src/screens/User/UserRoles/UserRolesList.jsx]: I wonder if this translates accurately.  I also can't tell that it does anything.  It probably can be removed.

**Review by AlexSCorey — CHANGES_REQUESTED** [review]
Just a few comments and some commented out code to be removed.

**Comment by keithjgrant** [comment]
> The User --> Roles list is confusing to me. I think this has been discussed and IIRC we have to lay it out this way due to limitations from from the api/pagination. However, it would be great if the Roles and Name column were switched because it seems that the roles are actually the subject of this list instead of the name of the resource the roles are applied to.

Yeah, this needs some thinking to (at the very least) line up with the current mockups better... but this matches prior behavior with the DataList, so that's something to tackle in another PR eventually

**Review by AlexSCorey — APPROVED** [review]
Looks good to me

**Comment by tiagodread** [comment]
@keithjgrant The user logged can see the tab `Token` when he clicks on his username, looks like we are missing this list

image

When the user click on `Tokens` the `Tab` is not being displayed
image

image

**Comment by tiagodread** [comment]
recheck

**Comment by tiagodread** [comment]
> @keithjgrant The user logged can see the tab `Token` when he clicks on his username, looks like we are missing this list
> 
> image
> 
> When the user click on `Tokens` the `Tab` is not being displayed
> image
> 
> image

@keithjgrant and I decided file a new issue for this problem here https://github.com/ansible/awx/issues/10092

**Review by tiagodread — APPROVED** [review]
Fixed broken e2e tests in 6299

**Pull request merged**